gradle-conjure/src/main/java/com/palantir/gradle/conjure/ConjureJavaLocalCodegenPlugin.java

```diff
@@ -118,13 +118,11 @@ private static void createGenerateTask(
 
         Provider<File> conjureIrFile = extractConjureIr.map(
                 irTask -> new File(irTask.getDestinationDir(), project.getName() + ".conjure.json"));
-        project.getTasks()
-                .named("configureProductDependencies", ConfigureProductDependenciesTask.class)
-                .configure(task -> {
-                    task.setProductDependencies(
-                            conjureIrFile.map(ConjureJavaLocalCodegenPlugin::extractProductDependencies));
-                    task.dependsOn(extractConjureIr);
-                });
+
+        project.getExtensions()
+                .getByType(RecommendedProductDependenciesExtension.class)
+                .getRecommendedProductDependenciesProvider()
```

Comment:
This provider is used as the source of truth in a couple places, previously we only updated one of its outputs